src/dom/node.ts

```diff
@@ -102,7 +102,7 @@ export class Node extends EventTarget {
       this._setOwnerDocument(newParent.#ownerDocument);
 
       // Add parent chain to ancestors
-      let parent: Node | null = newParent;
+      const parent: Node | null = newParent;
```

Comment:
Please revert this whole file, it is unrelated to the PR. If you want to submit this, please do it as a separate PR.